interface/src/app/app.js

```diff
@@ -19,15 +20,37 @@ angular.module( 'adage', [
   $resourceProvider.defaults.stripTrailingSlashes = false;
 }])
 
-.run( function run () {
+.run( function run ( $state, $location ) {
+
+    // Will only redirect if there is no state AND the url is not a
+    // route for tribe_client
+    String.prototype.startsWith = function(prefix) {
+        return this.slice(0, prefix.length) == prefix;
+    };
+
+    if ($state['current']['name'] === '') {
+      if (window.location.pathname.startsWith('/tribe_client')) {
```

Comment:
@rzelayafavila : Can there be a reusable angular app associated with tribe_client that provides the interface functionality? Thinking about how it can be modular across servers but also provide the functionality that you describe.